### ISSUE TITLE
Pandas 0.20.0 breaks arctic (get_timezone)

### ISSUE THREAD

**Issue by pquan** [manahl/arctic#350]
Trying to install arctic in a clean virtual env will pull down the latest pandas 0.20.0
This causes an issue as get_timezone is missing from pandas.tslib in version 0.20.0
The quick fix is to manually uninstall pandas and install 0.19.2

```
/home/test/pit/lib/python3.5/site-packages/arctic/store/_pandas_ndarray_store.py:6: FutureWarning: The pandas.tslib module is deprecated and will be removed in a future version.
  from pandas.tslib import Timestamp, get_timezone
Traceback (most recent call last):
  File "atest.py", line 1, in <module>
    from arctic import Arctic
  File "/home/test/pit/lib/python3.5/site-packages/arctic/__init__.py", line 6, in <module>
    from .store._pandas_ndarray_store import PandasDataFrameStore, PandasSeriesStore, PandasPanelStore
  File "/home/test/pit/lib/python3.5/site-packages/arctic/store/_pandas_ndarray_store.py", line 6, in <module>
    from pandas.tslib import Timestamp, get_timezone
ImportError: cannot import name 'get_timezone'

```

**Comment by bmoscon** [comment]
for now I'll probably just change the setup script to enforce a version less than or equal to 0.19.2 while I look for a workaround that it forwards and backwards compatible.

**Comment by bmoscon** [comment]
fixed with #353
